server/api/screenshot/screenshotController.js

```diff
@@ -2,6 +2,13 @@ var Screenshot = require('./screenshotModel');
 var User = require('../user/userModel');
 var takeScreenshot = require('../../screenshotCapture/script.js');
 
+exports.retrieveAll = function(req, res, next){
```

Comment:
Not that it's really implemented, but I think we should filter by the privacy setting. As long as it defaults to true right now in the schema, adding the line won't negatively impact anything